Variables.md

```diff
@@ -1,6 +1,6 @@
 # OpenIO SDS configuration
 
-`oio-sds` allows you to alter a lot of configuration at compile-time as well at runtime. We use a minimal generation tool for a set of variables that can be modified at runtime, and whose default value can be changed at the compile-time. Those variabes are described [here](./Variables.md)
+`oio-sds` allows you to alter a lot of configuration at compile-time as well at runtime. We use a minimal generation tool for a set of variables that can be modified at runtime, and whose default value can be changed at the compile-time. Those variables are described [here](./Variables.md)
```

Comment:
This file is generated by the `./confgen.py github conf.json` command which uses the `Variables.md.inc` file to add the header. We should fix the typo in the `Variables.md.inc` file.